cmd/edit/edit.go

```diff
@@ -0,0 +1,45 @@
+package edit
+
+import (
+	"fmt"
+	"io/ioutil"
+	"encoding/json"
+
+	"../../utils"
```

Comment:
Actually, an absolute path should be better. I think you can tell but I am a fledgling when it comes to GoLang.